sdcm/sct_runner.py

```diff
@@ -51,6 +50,14 @@ def instance_type(test_duration):
             return "r5.large"
         return "t3.large"  # has 7h 12m CPU burst
 
+    @staticmethod
+    def instance_root_disk_size(test_duration):
+        INSTANCE_ROOT_DISK_SIZE = 35  # GB
```

Comment:
we use upper case names for global variables only...
`INSTANCE_ROOT_DISK_SIZE` -> `instance_root_disk_size`

Reply:
instance_root_disk_size is same as the function name, I renamed it to disk_size.